src/Elastic.Apm.AspNetCore/Helpers/AgentConfigHelper.cs

```diff
@@ -0,0 +1,36 @@
+using System;
```

Comment:
Nit: some unused `using`s here.

Reply:
This file and all the other files I edited are now cleaned-up.